lib/ast.ts

```diff
@@ -15,6 +15,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/// <reference path="../types/node.d.ts" />
+require('source-map-support').install();
+
```

Comment:
This is kind of gross....why do you have to pollute a source file in order to get a source map?